week1/index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+
+<head>
+    <meta charset="utf-8">
+    <title>Make Your Home Great Again,Home Office</title>
+    <meta name="author" content="Ekip Kalir">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="main.css">
+    <script src="https://kit.fontawesome.com/f32ec08315.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <div class="nav-bar-top">
+        <header>
+            <ul>
+                <li><a href="#company-details">BlogX</a></li>
+                <li><a href="#articles-whole">Content</a></li>
+                <li><a href="#contact-details">Contact</a></li>
+                <li style="float:right"><a class="active" href="#summary">About</a></li>
```

Comment:
I see this inline style. You can't sneak them past me!

Reply:
I really resist not to use inline, however in some areas I just could not do that. Let me work on them.